lib/ui/cards/scoreboard_card.dart

```diff
@@ -1,63 +1,175 @@
-                mainAxisAlignment: MainAxisAlignment.spaceEvenly,
-                children: [
-                  Column(
-                    children: [
-                      Text(widget.model.daysInStudy.toString(),
-                          style: scoreNumberStyle.copyWith(
-                              color: Theme.of(context).primaryColor)),
-                      Text(locale.translate('cards.scoreboard.days'),
-                          style: scoreTextStyle.copyWith(
-                              color: Theme.of(context).primaryColor)),
-                    ],
+              Expanded(
+                child: shrinkOffset < offsetForShrink
+                    ? Column(
+                        mainAxisAlignment: MainAxisAlignment.center,
+                        children: childrenDays,
+                      )
+                    : Row(
+                        mainAxisAlignment: MainAxisAlignment.center,
+                        children: childrenDays,
+                      ),
+              ),
+              // A vertical divider line with rounded corners that spans from 10% to 90% of the height
+              Expanded(
+                flex: 0,
+                child: Container(
+                  height: calculateSize(
+                      shrinkOffset, minExtent * 0.6, maxExtent * 0.6),
+                  width: 2,
+                  decoration: BoxDecoration(
+                    color: Theme.of(context).dividerColor,
+                    borderRadius: BorderRadius.circular(100),
                   ),
-                  SizedBox(
-                      height: 66,
-                      child: VerticalDivider(
-                        color: Theme.of(context).primaryColor,
-                        width: 15,
-                      )),
-                  Column(
-                    children: [
-                      Text(widget.model.taskCompleted.toString(),
-                          style: scoreNumberStyle.copyWith(
-                              color: Theme.of(context).primaryColor)),
-                      Text(locale.translate('cards.scoreboard.tasks'),
-                          style: scoreTextStyle.copyWith(
-                              color: Theme.of(context).primaryColor)),
-                    ],
-                  )
-                ],
+                ),
               ),
+              Expanded(
+                child: shrinkOffset < offsetForShrink
+                    ? Column(
+                        mainAxisAlignment: MainAxisAlignment.center,
+                        children: childrenTasks,
+                      )
+                    : Row(
+                        mainAxisAlignment: MainAxisAlignment.center,
+                        children: childrenTasks,
+                      ),
+              )
             ],
           );
         },
       ),
     );
   }
+
+  // A simple function that returns the font size from the scoreNumberStyle, but increasingly smaller when scrolling down.
```

Comment:
You could just add this information in the function name then, rather then giving it a generic name, like calculateSize, name it something that will tell the reader what it is supposed to be